source/index.html

```diff
@@ -1,12 +1,169 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
+<html class="page" lang="ru">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+  <header class="main-header">
+    <a class="main-header__logo" href="index.html">
+      <img class="main-header__logo-image" src="img/logos/logo.svg" width="33" height="38"
+        alt="Логотип Cat Energy картинка рыжего кота в плаще.">
+    </a>
+    <a class="main-header__logo" href="index.html">
+      <img class="main-header__logo-text" src="img/logos/text-logo.svg" width="101" height="18"
+        alt="Логотип Cat Energy текстовая надпись.">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню.</span>
+      </button>
+      <ul class="main-nav__list site-list">
+        <li class="site-list__item site-list__item--active">
+          <a>
+            Главная
+          </a>
+        </li>
+        <li class="site-list__item">
+          <a href="catalog.html">
+            Каталог продукции
+          </a>
+        </li>
+        <li class="site-list__item">
+          <a href="form.html">
+            Подбор программы
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="offer">
+      <h1 class="offer__title">Функциональное<br>питание для котов</h1>
+      <span class="offer__intro">Занялся собой? займись котом!</span>
```

Comment:
```suggestion
      <p class="offer__intro">Занялся собой? займись котом!</p>
```

почему `p` а не `span`?